controllers/v1/programsController.js

```diff
@@ -641,4 +642,86 @@ module.exports = class Programs extends Abstract {
     })
   }
 
+    /**
+  * @api {post} /assessment/api/v1/programs/listByIds List programs by ids
+  * @apiVersion 1.0.0
+  * @apiName List programs by ids
+  * @apiGroup Program
```

Comment:
@aman-tunerlabs - I believe this has to internal access token accessible only and also the projection fields should come here, ideally not required fields should not be sent back in response